### ISSUE TITLE
 ModuleNotFoundError: No module named 'toml' when installing extension package with anything other than conda

### ISSUE THREAD

**Issue by reeceomahoney** [isaac-sim/IsaacLabExtensionTemplate#42]
When running the command `python -m pip install -e exts/ext_template` from the readme (with the extension name changed) the error in the title is thrown if anything other than conda is used for the virtual environment. I have a suspicion that this is due to the different way conda is handled in the IsaacLab install script. Wondering if anyone would know how to fix this?

**Comment by Ozzey** [comment]
Hello! I am facing the same problem. Did you manage to fix it?

**Comment by reeceomahoney** [comment]
Nope, I have just switched to using conda. Please let me know if you manage to find a fix

**Comment by tianzong-cheng** [comment]
Hi! Have you fixed the problem? Looks like `python setup.py develop` can work on my side.

**Comment by reeceomahoney** [comment]
Is this while using something other than conda as your env manager?

**Comment by tianzong-cheng** [comment]
> Is this while using something other than conda as your env manager?

I use Python venv

**Comment by reeceomahoney** [comment]
This worked for me as well, thanks!

**Comment by reeceomahoney** [comment]
I have come up with a proper fix for this issue to allow the pip install command to work. In exts/ext_template, on the same level as setup.py, add another pyproject.toml file with the following contents:

```
[build-system]
requires = ["setuptools", "wheel", "toml"]
build-backend = "setuptools.build_meta"
```

This allows toml to be found as a built dependency.
